mne/channels/channels.py

```diff
@@ -559,8 +559,9 @@ def drop_channels(self, ch_names):
 
         Parameters
         ----------
-        ch_names : list
-            The list of channels to remove.
+        ch_names : str | list
+            Name of the channel to remove, or the list of channels to
+            remove.
```

Comment:
>It seems there is some error for dropping nonexistent channels already though, it's just not triggered by my specific mistake. Will investigate.

My mistake: a test that should check for dropping channels without preloading was failing.